src/angular.headroom.js

```diff
@@ -1,37 +1,37 @@
-(function(angular) {
-
-  if(!angular) {
-    return;
```

Comment:
I just think it's a bit stupid to include a Directive but not include Angular. They'll get an error angular is not defined if they're not included Angular anyway.